### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by kucherenkodmytro**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+        [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "passed" if len(errors) == 0 else "failed"
+    }
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [format_single_linter_file
```
Comment (Grayder0152) [inline app/main.py:21]: Why did you add new line afte function name ?

File: app/main.py [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
+            (file_path=key, errors=value)
+            for key, value in linter_report.items()]
```
Comment (Grayder0152) [inline app/main.py:23]: Do not use `key` and `value` names. Create  more accurate names.

**Review by Grayder0152 — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -18,6 +18,5 @@ def format_single_linter_file(file_path: str, errors: list) -> dict:
 
 
 def format_linter_report(linter_report: dict) -> list:
-    return [format_single_linter_file
-            (file_path=key, errors=value)
-            for key, value in linter_report.items()]
+    return [format_single_linter_file(file_path=file, errors=status)
```
Comment (vasylhnatiuk) [inline app/main.py:21]: would be better to use:
```
return [
  format_single_linter_file(file_path=file, errors=status)
              for file, status in linter_report.items()
  ]
```

**Review by vasylhnatiuk — APPROVED** [review]
Good job

**Review by vasylhnatiuk — APPROVED** [review]

**Review by vasylhnatiuk — APPROVED** [review]
